tools/odbc/CMakeLists.txt

```diff
@@ -23,4 +34,8 @@ add_library(
 
 set_target_properties(duckdb_odbc PROPERTIES DEFINE_SYMBOL "DUCKDB_ODBC_API")
 
-target_link_libraries(duckdb_odbc duckdb_static)
+if(LINK_LIB_LIST)
+  target_link_libraries(duckdb_odbc ${LINK_LIB_LIST} duckdb_static)
```

Comment:
Since you initialize `LINK_LIB_LIST` to empty you shouldn't need this if no?

Reply:
Removed.